localstack/ext/java/src/main/java/cloud/localstack/Localstack.java

```diff
@@ -283,6 +307,24 @@ protected void setupInfrastructure() {
         }
     }
 
+    private void ensureJavaFilesRefreshed() {
+        String[] cmdPrepareJava = new String[]{"make", "-C", CURRENT_DEV_DIR
+                , "prepare-java-tests-infra-jars"};
+        exec(true, cmdPrepareJava);
+        Path localstackUtilsFatJar = Paths.get(CURRENT_DEV_DIR,"localstack", "infra", "localstack-utils-fat.jar");
```

Comment:
nitpick: we could assign `Paths.get(CURRENT_DEV_DIR, "localstack", "infra")` to a local variable and use it for the other paths, to avoid duplication in this method.

Reply:
Added